composer.json

```diff
@@ -44,6 +44,7 @@
         "league/csv": "9.23.0",
         "league/omnipay": "3.2.1",
         "mobiledetect/mobiledetectlib": "4.8.09",
+        "moneyphp/money": "4.7.0",
```

Comment:
Looks like a neat library. I did note in their README on github that 'This library requires the [BCMath PHP extension](https://www.php.net/manual/en/book.bc.php)', so would rec researching that before bringing this in.

Reply:
will have to add bcmath extenstion to the dockers, https://github.com/openemr/openemr-devops/pull/428, and then add it as a required package on the installation/upgrade page